src/Control/Email/Email.php

```diff
@@ -794,31 +816,40 @@ public function setPlainTemplate($template)
     }
 
     /**
+     * @deprecated 4.12.0 Will be removed without equivalent functionality
+     *
      * @param array $recipients
      * @return $this
      */
     public function setFailedRecipients($recipients)
     {
+        Deprecation::notice('4.12.0', 'Will be removed without equivalent functionality');
         $this->failedRecipients = $recipients;
 
         return $this;
     }
 
     /**
+     * @deprecated 4.12.0 Will be removed without equivalent functionality
+     *
      * @return array
      */
     public function getFailedRecipients()
     {
+        Deprecation::notice('4.12.0', 'Will be removed without equivalent functionality');
```

Comment:
https://github.com/silverstripe/silverstripe-framework/pull/10494 still has a private `$failedRecipients` array - why not keep this method?

Reply:
Shouldn't be there any more, have deleted from that PR